components/tabs/style/index.less

```diff
@@ -165,7 +165,7 @@
       }
     }
 
-    &:hover {
+    &.ant-tab-hover {
```

Comment:
Please do not change any styles except `patch.less`, because these files are synced from `ant-design`.We can put it to `patch.less`.
Of course, if this style does not affect the final experience, we can also ignore it for now, after that we can approve and merge this PR :)

Reply:
We can add the new class in `patch.less`, but if we leave this as it is now (generated by `ant-design`) with `&:hover` it will still remain broken because this CSS on hover will trigger the old behavior and newly created directive will do nothing.

Reply:
Should I add this class in `patch.less` and open PR in the `ant-design` repo or leave it as it is?

Reply:
You need to add these changes to patch.less like other components, after that we can merge this PR :)

Reply:
Problem is that in `index.less` we have 
```
&:hover {
      color: @tabs-hover-color;
    }
```
so it will always override the added class from `patch.less`